### PR TITLE
Enable UFMT format on test/test_utils.py

### PR DESCRIPTION
**Pull request by zeshengzong**
Fixes some files in #123062

Run lintrunner on files:
test/test_utils.py

```bash
$ lintrunner -a --take UFMT --all-files
ok No lint issues.
Successfully applied all patches.

### CODE REVIEW

File: test/test_utils.py [hunk 1]
```diff
@@ -534,52 +585,68 @@ def test_single_keep(self):
...
         import subprocess
 
-        p = subprocess.Popen(command, stdout=subprocess.PIPE,  # noqa: P204
-                             stderr=subprocess.PIPE, shell=True)
+        p = subprocess.Popen(
+            command,
+            stdout=subprocess.PIPE,  # noqa: P204
+            stderr=subprocess.PIPE,
+            shell=True,
+        )
```
Comment (shink) [inline test/test_utils.py:649]: the location of `noqa` may be wrong

**Comment by ezyang** [comment]
is clean

File: test/test_utils.py (hunk 1 above)
Comment (hippocookie) [reply test/test_utils.py]: Maybe it's unnecessary, let me try to remove it.

**Review by ezyang — APPROVED** [review]

**Pull request closed without merging**